modules/olmv1-creating-a-cluster-role.adoc

```diff
@@ -0,0 +1,334 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="olmv1-creating-a-cluster-role_{context}"]
+= Creating a cluster role for an extension
+
+You must review the cluster service version (CSV) and manifests of an extension carefully to define the required role-based access controls (RBAC) for the extension you want to install. You must create a cluster role by copying the required RBAC from the CSV to the new manifest.
+
+[IMPORTANT]
```

Comment:
I don't know if this callout should should a NOTE, IMPORTANT, or a WARNING. It has parts of all three, IMO. It warns about the consequences of not using the permissions in production clusters but also gives guidance on how to test the permissions. 

I wonder if this section could be reworked a little. The language of `You must` made me think that part of those two intro sentences could be placed in the important and perhaps this could be pulled out of the callout and discussed as the intro.

Reply:
Good point. I'll use the TIP admonition instead. That is probably the right format.